llvm/tools/llvm-exegesis/lib/SnippetGenerator.h

```diff
@@ -55,6 +55,7 @@ class SnippetGenerator {
 public:
   struct Options {
     unsigned MaxConfigsPerOpcode = 1;
+    std::optional<int64_t> InitImmValue;
```

Comment:
Is this used anywhere?

Reply:
Now it really isn't used. We made it with expansion in mind. Deleted in this commit